src/code42cli/cmds/securitydata_output_formats.py

```diff
@@ -0,0 +1,100 @@
+from datetime import datetime
```

Comment:
filename `securitydata_...` doesn't seem apt, it should be `CEF_...`  because this format even though today is supported only for security data, it can be supported elsewhere. or if we want proper modularisation then `cmd/output_formats/CEF.py`

Reply:
This specific module only has to do with file events.  Mapping to CEF is very business-logic-y and will vary a lot with each data set.

Reply:
Also contains `file_events_output_format` which could potentially include other variances of file event-realted output formats

Reply:
Then its fine.